kotlinx.fuzz.api/src/main/kotlin/kotlinx/fuzz/KFuzzConfig.kt

```diff
@@ -72,6 +79,18 @@ class KFuzzConfigImpl private constructor() : KFuzzConfig {
         toString = { it.inWholeSeconds.toString() },
         fromString = { it.toInt().seconds },
     )
+    override var runMode: RunMode by KFuzzConfigProperty(
+        "kotlinx.fuzz.runMode",
+        defaultValue = RunMode.REGRESSION_FUZZING,
+        toString = { it.toString() },
+        fromString = { RunMode.valueOf(it.uppercase()) },
+    )
+    override var reproducerPath: Path by KFuzzConfigProperty(
+        "kotlinx.fuzz.reproducerPath",
+        defaultValue = Paths.get("."),
+        toString = { it.absolutePathString() },
+        fromString = { Paths.get(it) },
```

Comment:
`defaultValue = Path(".")` 

`fromString = { Path(it).absolute() }`

You might need to manually write `import kotlin.io.path.Path`